lte/gateway/c/core/oai/test/amf/test_amf_encode_decode.cpp

```diff
@@ -1465,4 +1465,56 @@ TEST(test_optional_pdu, test_pdu_session_accept_optional) {
   sm_free_protocol_configuration_options(&msg_accept_pco);
 }
 
+TEST(test_ambr, test_ambr_covert) {
```

Comment:
covert -> convert

Also I don't think `test_amf_encode_decode.cpp` is the right place to test common utility functions. cc @panyogesh

Reply:
I changed the location for that unit test function.

Reply:
I changed the location for it.